Base/usr/share/man/man1/unzip.md

```diff
@@ -0,0 +1,25 @@
+## Name
+
+**unzip** - extract files in a ZIP archive
```

Comment:
Shouldn't be bold.

And perhaps *from* a ZIP archive?

Reply:
You're right, I was looking at the wrong man when I took the format to write it. Also yeah, better with *from*. Thanks